doc/nrf/app_dev/bootloaders_dfu/dfu_tools_mcumgr.rst

```diff
@@ -0,0 +1,527 @@
+.. _dfu_tools_mcumgr:
+
+DFU management tools
+####################
```

Comment:
This topic should go and it should start directly with the command-line tool one

Reply:
But shouldn't the command-line tool section have the same heading level as image and statistic management? If so, then we need to have a main heading to correctly reflect the structure and to align it with ncs documentation standards

Reply:
This doesn't list the tools, the tools are listed upstream, this lists one specific tool